README.md

```diff
@@ -7,24 +7,49 @@ The PyFingerprint library allows to use ZhianTec ZFM-20, ZFM-60, ZFM-70 and ZFM-
 **Note:** The library is inspired by the C++ library from Adafruit Industries:
 <https://github.com/adafruit/Adafruit-Fingerprint-Sensor-Library>
 
-## Package building on Debian
+## Installation
+
+There are two ways of installing PyFingerprint: Installation of the stable or latest version. The stable version is distributed through the PM Code Works APT repository and is fully tested but does not contain the latest changes.
+
+### Installation of the stable version
+
+Add PM Code Works repository
+
+* Debian 9:
+
+    `~# echo "deb http://apt.pm-codeworks.de stretch main" | tee /etc/apt/sources.list.d/pm-codeworks.list`
+
+* Debian 10:
+
+    `~# echo "deb http://apt.pm-codeworks.de buster main" | tee /etc/apt/sources.list.d/pm-codeworks.list`
+
+Add PM Code Works signing key
 
-Install the packages for building:
+    ~# wget -qO - http://apt.pm-codeworks.de/pm-codeworks.de.gpg | apt-key add -
+    ~# apt-get update
```

Comment:
imho we should change from "apt-get" to "apt" commands. Also instead of `~#` and `~$` we should just write `#` and `$`